app/experimenter/nimbus-ui/src/components/PageExperimentNotFound/index.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Link } from "@reach/router";
+import React from "react";
+import { BASE_PATH } from "../../lib/constants";
+import AppLayout from "../AppLayout";
+
+const ExperimentNotFound = ({ slug }: { slug: string }) => (
+  <AppLayout testid="not-found">
+    <section className="text-center">
+      <h1 className="h2">Experiment Not Found</h1>
+      <p className="pt-3">{`The experiment with slug "${slug}" could not be found. ☹️`}</p>
```

Comment:
What, we couldn't afford the real emoji?!
<img width="131" alt="Screen Shot 2020-10-27 at 2 41 32 PM" src="https://user-images.githubusercontent.com/6392049/97346989-8e686500-1862-11eb-85b3-01756064e77b.png">

Reply:
Huh, for some reason in VSCode it shows like this but in the app and Storybook I see the real emoji. Are you seeing it look like this outside of VSCode? 😂

Reply:
yeah!

Reply:
Works on my machine™️

Reply:
RUDE

Reply:
swap it out for Mr. Coconut